privacyidea/lib/tokens/emailtoken.py

```diff
@@ -268,7 +268,7 @@ def create_challenge(self, transactionid=None, options=None):
                                                         "{0!s}_{1!s}".format(self.get_class_type(),
                                                                              ACTION.CHALLENGETEXT),
                                                         options) or _("Enter the OTP from the Email:")
-        attributes = {'state': transactionid}
+        attributes = {'attributes': {'state': transactionid}}
```

Comment:
Please rename the variable to ``reply_dict`` analogous to the other tokens.

Reply:
Done in db7cb708